config/overlays/test/qpext_image_patch.yaml

```diff
@@ -0,0 +1,2 @@
+data:
+  queueSidecarImage: kserve/qpext:latest
```

Comment:
We can remove this file now

Reply:
I use this in the e2e test

Reply:
changed it to using a the patch command with json instead of a file